config.rb

```diff
@@ -18,6 +24,37 @@
 tag = JSON.parse(response)['tag_name']
 set :latest_release_tag, tag
 
+# Populate docs pages
+docs_source_dir = 'kubeless-src'
+checkout_commit = '1c592cdc55bf009dc58773a042e42d70085e930b' # Use tag or a specific commit if necessary
```

Comment:
I don't like to hardcode a commit or tag in the code. Why don't update the submodule manually to update the documentation?